src/utils/modifiers.utils.ts

```diff
@@ -0,0 +1,248 @@
+): Promise<{ success: boolean; startOffset?: number }> => {
+	return new Promise((resolve) => {
+		// Access the active text editor
+		const activeEditor = vscode.window.activeTextEditor;
+
+		// Check if there is an active text editor
+		if (activeEditor) {
+			// Initialize startOffset to -1 (to find the start of imports)
+			let startOffset = -1;
+
+			// Use the edit method to modify the document
+			activeEditor
+				.edit((textEditor) => {
+					// For loop on imports (array of strings)
+					for (const searchString of imports) {
+						// Find the startOffset (start of imports)
+						const currentOffset = activeEditor.document
+							.getText()
+							.indexOf(searchString);
+						if (
+							startOffset === -1 ||
+							(currentOffset !== -1 && currentOffset < startOffset)
+						) {
+							startOffset = currentOffset;
+						}
+						if (currentOffset !== -1) {
+							// Find end offset of a search string
+							const endOffset = currentOffset + searchString.length;
+							// Find start position of imports
+							const startPosition = activeEditor.document.positionAt(currentOffset);
+							// Find end position of imports
+							const endPosition = activeEditor.document.positionAt(endOffset);
+
+							// Loop from start line to end line of imports
+							for (
+								let lineNumber = startPosition.line;
+								lineNumber <= endPosition.line;
+								lineNumber++
+							) {
+								// Create line object for each line and delete it
+								const lineObject = activeEditor.document.lineAt(lineNumber);
+								const lineRange = lineObject.rangeIncludingLineBreak;
+
+								// Delete text of Line Range
+								textEditor.delete(lineRange);
+							}
+						}
+					}
+				})
+				.then(() => {
+					// Resolve the promise with the deletion result and start offset
+					resolve({ success: true, startOffset });
+				});
+		} else {
+			// If there is no active text editor, resolve the promise with false
+			resolve({ success: false });
+		}
+	});
+};
+
+/** To delete the correct import comments ( // CATEGORY // format ) */
+export const deleteCategoryComments = (): Promise<{ success: boolean }> => {
+	return new Promise((resolve) => {
+		// Access the active text editor
+		const activeEditor = vscode.window.activeTextEditor;
+
+		// Check if there is an active text editor
+		if (activeEditor) {
+			// Use the edit method to modify the document
+			activeEditor
+				.edit((textEditor) => {
+					// Iterate through lines to check for comments of imports
+					for (
+						let lineNumber = 0;
+						lineNumber <= activeEditor.document.lineCount - 1;
+						lineNumber++
+					) {
+						const lineObject = activeEditor.document.lineAt(lineNumber);
+						// Regular expression for comments
+						const commentMatch = lineObject.text.match(/\/\/.*?\/\/.*?/);
+
+						// Check if the line contains a comment
+						if (commentMatch) {
+							// Iterate through each import type
+							for (const importType of importCategories) {
```

Comment:
use a different array for Deleting. Because that array will have the wrong spellings and stuff too